sentry_sdk/integrations/fastapi.py

```diff
@@ -62,6 +65,21 @@ def patch_get_request_handler():
 
     def _sentry_get_request_handler(*args, **kwargs):
         # type: (*Any, **Any) -> Any
+        dependant = kwargs.get("dependant")
+        if dependant and not asyncio.iscoroutinefunction(dependant.call):
+            old_call = dependant.call
+
+            def _sentry_call(*args, **kwargs):
```

Comment:
is setting the thread in fastapi still necessary if we already do it in `request_response` in starlette?